### PR TITLE
Add content for README and fix small bugs

### PR DESCRIPTION
**Pull request by DeLaGuardo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,569 @@
 
 > Expressions language for Clojure(Script) and JavaScript inspired by Microsoft Excel ™
 
+# Rationale
+
+In some applications, the lion's share of business logic is concentrated in dynamic expressions. Often they are not even part of the code base and are stored in a database, files, or somewhere in the cloud. And sometimes these expressions need to be accessed not by programmers. For example, managers or operators of call centers. Making them learn clojure or python will not be fair. This library is designed to combine the best of two completely different worlds: the most understandable syntax from Excel and the speed of Clojure.
```
Comment (DG4227) [inline README.md]: "need to be accessed by _non-technical personnel, like_ managers or call center operators. Making them learn clojure or python _is_ not fair."

File: README.md [hunk 2]
```diff
@@ -2,6 +2,569 @@
...
+
+## Clojure(Script)
+
+* **Leiningen** : `[io.xapix/axel-f "0.1.2"]`
+* **Boot**: `(set-env! :dependencies #(conj % [io.xapix/axel-f "0.1.2"]))`
+* **deps.edn**: `{:deps {io.xapix/axel-f {:mvn/version "0.1.2"}}}`
+
+## JavaScript
+
+[Please checkout documentation for JavaScript package](https://github.com/xapix-io/axel-f/tree/master/release-js)
```
Comment (DG4227) [inline README.md]: "checkout _the_ documentation"

File: README.md [hunk 3]
```diff
@@ -2,6 +2,569 @@
...
+;;   2
+;;   [:FNCALL "AVERAGE" [[:VECTOR 4 5 6]]]
+;;   [:OBJREF "foo" "bar"]
+;;   [:OBJREF "foo" "baz" "*" "x"]]]
+
+```
+
+# Difference from Excel
+
+* No cell-references and reference operations.
```
Comment (DG4227) [inline README.md]: "_or_ reference operations"

File: README.md [hunk 4]
```diff
@@ -2,6 +2,569 @@
...
+* Excel errors replaced with Exceptions from Clojure(Script)
+
+# Object references
+
+In addition to the formula run function can accept execution context as a second argument. Context can be any valid Clojure(Script) object. In formula you can select the data from context by using object reference operators:
```
Comment (DG4227) [inline README.md]: "to _a_ formula, _the_ run function... In _the_ formula..."

File: README.md [hunk 5]
```diff
@@ -2,6 +2,569 @@
...
+
+* Dot reference operator for access nested data: `foo.bar.baz`
+* Array reference operator for access data in vector: `foo[*].bar`
+  * `foo[*][*]` we support nested vectors (vector of vectors of vectors ...)
+  * and objects in vectors `foo[*].bar[*].baz`
+
+# Data types
+
+- [x] Booleans as they known in excel (TRUE/FALSE). In addition axel-f understands `True/False/true/false`
```
Comment (DG4227) [inline README.md]: "as _in_ Excel"

File: README.md [hunk 6]
```diff
@@ -2,6 +2,569 @@
...
+- [x] Numbers (Integers, Floats, Exponential form)
+- [x] Strings in double or single quotes. (`'Some String'`, `"Some String"`)
+- [x] Arrays. Any data in curly brackets (`{1, 2 TRUE}`)
+- [ ] Date
+- [ ] Excel Error types
+
+# Operators
+
+Any expressions can be used as a operand for any operator. axel-f has the same operator precedence as in Excel. To change a precendence of operators you can use round brackets (`(2+2)*2` => 8)
```
Comment (DG4227) [inline README.md]: Any _expression_ can be used as _an_

File: README.md [hunk 7]
```diff
@@ -2,6 +2,569 @@
...
+- [x] MIN
+- [x] MAX
+- [x] CONCATENATE
+- [x] IF
+- [x] AVERAGE
+- [x] ROUND
+- [x] AND
+- [x] OR
+
+In addition we have special function for accessing the data in context: `OBJREF(arg1, [arg2...])`
```
Comment (DG4227) [inline README.md]: special _functions_

**Comment by DeLaGuardo** [comment]
thanks!

**Pull request merged**